front-end/src/components/App.tsx

```diff
@@ -4,13 +4,14 @@ import { UploadModal } from './modal/upload/UploadModal';
 import { HelpModal } from './modal/help/helpModal';
 import { DownloadModal } from './modal/download/DownloadModal';
 import { useEffect, useState } from 'react';
-import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { BrowserRouter, HashRouter, Routes, Route } from 'react-router-dom';
 import Topbar from './Topbar/Topbar';
 import Views from './views/Views/Views';
 import $ from 'jquery';
 import { MAX_VIEWS } from './views/viewsConfig';
 import Chart from './views/Chart/Chart';
 import MapChart from './map/MapChart/MapChart';
+import { isElectron } from '@lib/apiUtils';
```

Comment:
All this isElectron stuff needs to be thought out a bit more. The changes to the codebase that change how the dev has to work, just for electron to function is quite annoying + hard to understand why its like that, what changes might break it, etc. If a HashRouter would work for both, we can just use that. If there are downsides, we should rethink our approach

Reply:
moved to be handled in utils file